server/managers/BackupManager.js

```diff
@@ -82,7 +83,7 @@ class BackupManager {
       return res.status(500).send('Invalid backup file')
     }
 
-    const tempPath = Path.join(this.BackupPath, backupFile.name)
+    const tempPath = fileUtils.sanitizeFilename(Path.join(this.BackupPath, backupFile.name))
```

Comment:
```suggestion
    const tempPath = Path.join(this.BackupPath, fileUtils.sanitizeFilename(backupFile.name))
```
This function was just written for filenames